src/pages/storage/forms/UploadVolumeBackupFileForm.tsx

```diff
@@ -0,0 +1,286 @@
+      clusterMember: clusterMembers?.[0]?.server_name ?? "",
+    },
+    validateOnMount: true,
+    validationSchema: Yup.object().shape({
+      name: Yup.string()
+        .test(
+          ...testDuplicateStorageVolumeName(
+            project?.name || "",
+            "custom",
+            volumeNameAbort,
+          ),
+        )
+        .optional(),
+    }),
+    onSubmit: handleUpload,
+  });
+
+  const pool = pools.find((pool) => pool.name == formik.values.pool);
+  const isCMSDriver = isClusterLocalDriver(pool?.driver || "");
+
+  const handleCloseModal = useCallback(() => {
+    uploadAbort?.abort();
+    setUploadState(null);
+    setUploadAbort(null);
+    formik.resetForm();
+    close();
+    notify.clear();
+  }, [uploadAbort, formik.resetForm, close, notify]);
+
+  const changeFile = async (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange } = formik.getFieldProps("volumeFile");
+    onChange(e);
+
+    if (e.currentTarget.files) {
+      const file = e.currentTarget.files[0];
+      await formik.setFieldValue("volumeFile", file);
+
+      if (!defaultVolumeName) {
+        const volumeName = fileToSanitisedName(file.name, "-import");
+        await formik.setFieldValue("name", volumeName);
+
+        // validate volume name
+        await formik.validateField("name");
+        void formik.setFieldTouched("name", true, true);
+        if (!formik.errors.name) {
+          formik.setFieldError("name", undefined);
+        }
+      }
+    }
+  };
+
+  const noFileSelectedMessage = !formik.values.volumeFile
+    ? "Please select a file before adding custom configuration."
+    : "";
+
+  return (
+    <>
+      <Form
+        onSubmit={formik.handleSubmit}
+        className={classnames({ "u-hide": uploadState })}
+      >
+        <Input
+          id="volume-file"
+          name="volumeFile"
+          type="file"
+          accept=".tar, application/gzip, application/x-bzip, application/x-xz, application/x-lzma, application/x-squashfs, application/x-qcow2, application/zstd"
+          label="LXD backup archive (.tar.gz)"
+          onChange={(e) => void changeFile(e)}
+        />
+        <Input
+          {...formik.getFieldProps("name")}
+          id="name"
+          type="text"
+          label="New volume name"
+          placeholder="Enter name"
+          error={formik.touched.name ? formik.errors.name : null}
+          disabled={!!noFileSelectedMessage}
+          title={noFileSelectedMessage}
+        />
+        <StoragePoolSelector
+          value={formik.values.pool}
+          setValue={(value) => {
+            void formik.setFieldValue("pool", value);
+          }}
+          selectProps={{
+            id: "volume-import-pool",
+            label: "Storage pool",
+            disabled: isLoading || !!noFileSelectedMessage,
+            title: noFileSelectedMessage,
+          }}
+        />
+        <ClusterMemberSelector
+          {...formik.getFieldProps("targetClusterMember")}
```

Comment:
```suggestion
          {...formik.getFieldProps("clusterMember")}
```
This is now updating an outdated field name.